### PR TITLE
Add win

### PR DESCRIPTION
**Pull request by godofredoc**
## Description

Adds the windows presubmit builder temporarily while we finish the migration to use shards.

## Related Issues

https://github.com/flutter/flutter/issues/63587

## Tests

N/A

## Checklist

Before you create this PR, confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X] I signed the [CLA].
- [X] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [X] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [X] I updated/added relevant documentation (doc comments with `///`).
- [X] All existing and new tests are passing.
- [X] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [X] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Did any tests fail when you ran them? Please read [Handling breaking changes].

- [X] No, no existing tests failed, so this is *not* a breaking change.
- [ ] Yes, this is a breaking change. *If not, delete the remainder of this section.*
   - [ ] I wrote a design doc: https://flutter.dev/go/template *Replace this with a link to your design doc's short link*
   - [ ] I got input from the developer relations team, specifically from: *Replace with the names of who gave advice*
   - [ ] I wrote a migration guide: https://flutter.dev/go/breaking-changes-template *Replace this with a link to a pull request that adds the migration guide to https://flutter.dev/docs/release/breaking-changes*

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Test Coverage]: https://github.com/flutter/flutter/wiki/Test-coverage-for-package%3Aflutter
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

**Review by keyonghan — APPROVED** [review]
You may need to rebase with master. It combines earlier commits (like the one changing prod builders).

**Comment by godofredoc** [comment]
> t combines earlier commits (like the one changing prod builders).

done

**Comment by Piinks** [comment]
Some of the luci builds timed out here. I've restarted them.

**Comment by godofredoc** [comment]
> Some of the luci builds timed out here. I've restarted them.

Replaced the monolithic version with the sharded versions and re-run tests.

**Comment by keyonghan** [comment]
Seems `Windows web_tests` is missing?

**Comment by godofredoc** [comment]
> Seems [`Windows web_tests`](https://github.com/flutter/infra/blob/master/config/framework_config.star#L289) is missing?

That's intentional they are currently failing once they pass I'll add them.

File: dev/try_builders.json [hunk 1]
```diff
@@ -24,6 +24,11 @@
          "name":"Linux web_tests",
          "repo":"flutter",
          "taskName":"web_tests"
+      },
+      {
+         "name":"Windows",
+         "repo":"flutter",
+         "taskName":"windows_bot"
```
Comment (keyonghan) [inline dev/try_builders.json:31]: shall this be added back to framwork_config.star in `infra` repo?

**Pull request merged**